db/migrate/20150705131449_add_type_to_reports.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToReports < ActiveRecord::Migration
+  def change
+    add_column :reports, :type, :string, :default => 'PuppetReport'
```

Comment:
I would prefer if there was a generic ConfigReport that has the general idea behind a config management system: system status, resources, results, etc, and allow config management platforms to extend it if necessary.  Does that make sense?

That would let puppet/salt/chef customize it, while still providing the base data to be the source of data for the host status, dashboard, etc.